gateway-core/components/org.wso2.carbon.gateway.core/src/test/java/org/wso2/carbon/gateway/core/config/WUMLListenerTest.java

```diff
@@ -163,6 +259,10 @@ public String getName() {
         }
 
         @Override
+        public void setName(String name) {
+        }
```

Comment:
Actually, I didn't put this. The method as there already, git identified it as removed old one and added same again. 
But anyway this is required here, because we are trying to create a sample mock object(class) by implementing OutboundEndpoint interface.
thanks,